### ISSUE TITLE
Potential bug in choropleth()?

### ISSUE THREAD

**Issue by vkchan** [python-visualization/folium#592]
Please see below:
[GeoJSON data not displaying in Python folium map](http://stackoverflow.com/questions/42107268/geojson-data-not-displaying-in-python-folium-map)

I tried to use choropleth() to plot the specified GeoJSON file, but all I got is a empty map of NYC. However I was able to use [MapShaper](http://mapshaper.org/) to display that GeoJSON file without any issue.

Is it a potential bug in choropleth()? 

Thanks!

**Comment by ocefpaf** [comment]
See my answer at http://stackoverflow.com/questions/42107268/geojson-data-not-displaying-in-python-folium-map/42633407#42633407 and my example with your data at: http://nbviewer.ipython.org/e1e2776e3ea69bae5b6e5449636ac57f

TL;DR your data is not a GeoJSON but rather a TopoJSON.

**Comment by vkchan** [comment]
Thanks a lot, I didn't realize it's a TopoJSON file, now it works perfectly.
